test/OrchardCore.Tests/Apis/SiteStartup.cs

```diff
@@ -22,11 +21,17 @@ public SiteStartup(TestSiteConfiguration configuration)
 
         public void ConfigureServices(IServiceCollection services)
         {
-            services.AddOrchardCms();
+            services.AddOrchardCms().AddOrchardCore()
```

Comment:
If we need to call `AddOrchardCore` after `AddOrchardCms` to be able to configure the features, then we need to add an overload to accept an `Action<OrchardCoreBuilder>` in `AddOrchardCms`.

Reply:
`AddOrchardCore()` has this ability.

Why not just letting `AddOrchardCms()` returning a builder as `AddOrchardCore()` do?

Reply:
It _could_ return an `OrchardCmsBuilder` but it should still accept an `OrchardCoreBuilder` as a custom action.

Reply:
Okay cool, as `AddOrchardCore()`, i will update it.

Reply:
For infos.

Just saw that the overload of `AddOrchardCore()` which accepts an action as been removed. But maybe useless here.